README.md

```diff
@@ -20,7 +20,7 @@ To run a Docker container
 
   Linux agent:
 
-    docker run jenkins/jnlp-slave -url http://jenkins-server:port <secret> <agent name>
+    docker run --init jenkins/jnlp-slave -url http://jenkins-server:port <secret> <agent name>
```

Comment:
I still think it would be good to have a Note saying why the --init flag is useful.

Reply:
Oh, I'm not good at docs wording.
> Note: `--init` is necessary for correct subprocesses handling (zombie reaping)

Is it ok?

Reply:
Yes, that's a good start. You could also link to the docker docs on `--init` so people can easily look further if desired.

Reply:
`docker run` docs don't provide detailed explanation either: `Run an init inside the container that forwards signals and reaps processes`.
I think it is expected that if you don't know what is "zombie reaping" you just google where do zombies come from.

Reply:
To be clear, I was not meant "you" literally, but someone who will read the docs.